.github/workflows/swagger.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish Swagger Specs
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle-
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Run Swagger Publisher
+        run: ./gradlew integration --tests uk.gov.hmcts.reform.ccd.documentam.config.SwaggerPublisher
+      - name: Commit to repository
+        run: |
+          mkdir swagger-staging
```

Comment:
Is there a scenario where this directory may already exist ?

 `mkdir -p swagger-staging` so it doesn't error if dir already exists?

Reply:
or we could delete the directory at the end ?

Reply:
github actions build creates a new workspace for every build and does clean up as well.

Reply:
ok great !